packages/entity-server/src/auth/middleware.ts

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
Still a lot of duplication in this file with `packages/report-server/src/auth/middleware.ts`

Personally I think it acceptable as most of this code is sort of boilerplate and there isn't much complex logic that is being duplicated, however happy to get push back on this.

Reply:
Could move into `@tupaia/server-boilerplate` once @tcaiger's PR hits dev

Reply:
I agree!